src/Renderer/RendererOpenGL.cpp

```diff
@@ -572,6 +572,7 @@ void RendererOpenGL::size(int w, int h)
 {
 	SDL_SetWindowSize(underlyingWindow, w, h);
 	setViewport(w, h);
+	setOrthoProjection(w, h);
```

Comment:
This commit is focused on adding a new method, which makes this update seem a bit out of place.